web/client/components/map/cesium/plugins/WMTSLayer.js

```diff
@@ -115,11 +116,19 @@ function wmtsToCesiumOptions(_options) {
     const queryParametersString = urlParser.format({ query: {...getAuthenticationParam(options)}});
     const cr = options.credits;
     const credit = cr ? new Cesium.Credit(creditsToAttribution(cr)) : '';
+    let headers;
+    if (options.security) {
+        const storedProtectedService = getCredentials(options.security?.sourceId) || {};
+        headers = {
+            "Authorization": `Basic ${btoa(storedProtectedService.username + ":" + storedProtectedService.password)}`
+        };
+    }
     return assign({
         // TODO: multi-domain support, if use {s} switches to RESTFul mode
         url: new Cesium.Resource({
             url: head(getURLs(isArray(options.url) ? options.url : [options.url], queryParametersString)),
-            proxy: proxy && new WMTSProxy(proxy) || new NoProxy()
+            proxy: proxy && new WMTSProxy(proxy) || new NoProxy(),
+            ...(headers)
```

Comment:
Looking at the doc, [headers](https://cesium.com/learn/ion-sdk/ref-doc/Resource.html?classFilter=Resource#headers) should be an object not spread inside the `Cesium.Resource` class, could you double check if it's working?